src/index.ts

```diff
@@ -242,6 +244,7 @@ export class Supertab {
     if (access.access) {
       return {
         access: {
+          contentKey: access.access.contentKey,
```

Comment:
Returning `contentKey` from access service in the response now.

Reply:
Is it necessary to keep backwards compatibility? My understanding from our conversation yesterday was that we simply make the param required and release a new major version.

Reply:
On second thought, I realize that we currently don't expose the `contentKey`(s) from client config, right? So in order to keep access checking possible with client config (non-experience) integrations, the new major version would have to either expose the `contentKey` from client config or make the param optional and default to the old behavior (like this PR suggests).

@tomasstark Does this match your reasoning or do you have something else in mind?

Reply:
@tomasstark Only now I carefully read the PR description again and saw this part:
>This is because we will be moving away from client config entirely soon so the plan is to introduce one major version with all related breaking changes.

Makes sense to me 👍

Reply:
😄 Good talk, @jmcodingde.